packages/remirror__extension-node-formatting/__tests__/node-formatting-extension.spec.ts

```diff
@@ -1,5 +1,96 @@
-import { extensionValidityTest } from 'jest-remirror';
+import { extensionValidityTest, renderEditor } from 'jest-remirror';
+import { BulletListExtension } from '@remirror/extension-list';
```

Comment:
Can we

```suggestion
import { BulletListExtension } from 'remirror/extensions';
```

To avoid the need for the new dependency in `package.json`?